NEWS.rst

```diff
@@ -47,6 +47,7 @@ possible, especially the following contributors:
 - Peter Cock
 - Sergio Valqui
 - Zheng Ruan
+- Yogesh Kulkarni (first contribution)
```

Comment:
Can you move your name up one (alphabetical by first name)?

Reply:
Done. Please have a look.